src/sage/schemes/affine/affine_space.py

```diff
@@ -883,21 +884,21 @@ def chebyshev_polynomial(self, n, kind='first', monic=False):
         EXAMPLES::
 
             sage: A.<x> = AffineSpace(QQ, 1)
-            sage: A.chebyshev_polynomial(5, 'first')
+            sage: A.chebyshev_polynomial(5, 'first')                                    # needs sage.schemes
             Dynamical System of Affine Space of dimension 1 over Rational Field
```

Comment:
Why didn't you include `src/sage/schemes/affine,projective,generic` to `sage.schemes`?

Reply:
It seems to me that the "cores" of schemes are excluded from `sage.schemes`.

Reply:
They are already shipped by the dependency distribution sagemath-categories; roughly because multivariate polynomials are also provided by that.

Reply:
OK for now.